lib/ReactCrop.js

```diff
@@ -222,9 +222,19 @@ class ReactCrop extends PureComponent {
     }
   }
 
+  getImageDimensions() {
+    if (this.props.renderComponent) {
+      return {
+        width: this.imageRef.offsetWidth,
```

Comment:
I haven't run it yet but what's this doing? I see you set the image width and height to 0, wondering what values this is giving?

Reply:
When a custom component is passed in, we set the `imageRef` to be a reference to the div wrapping that component. That way, we can get the width and height of the component itself. See [here](https://github.com/DominicTobias/react-image-crop/pull/274/files/88a290bedcff164448d8d1de6213fd9b5e27a66b#diff-0739dd7cc09fcb465f0ba0cf44b0aa8eR778)

Reply:
Ah got it